experimental/bigint.md

```diff
@@ -0,0 +1,7 @@
+# [BigInt](https://github.com/tc39/proposal-bigint)
+
+```js
+interface BigIntLiteral <: Literal {
+  bigint: string;
```

Comment:
Other `Literal`s use `value` as the property name.

Reply:
As far as I understand, the idea here is to have it similar to `RegExpLiteral` where `value` might or might not be representable on the target runtime so another field is needed to represent the parsing result in these cases.

Reply:
Actually, you were the one who suggested that in https://github.com/estree/estree/issues/169#issuecomment-329059842

Reply:
Ah, that's right, there's no `"type"` or anything distinguishing it from other `Literal`s. Got it. Okay, what should the `value` of these nodes be? Currently bigints are not allowed in the `value` field.

Reply:
You suggested to use a BigInt literal for `value`. For that we would need to amend the `Literal` interface (`extend interface Literal { type: string | boolean | null | number | RegExp | bigint;`) and add `value: bigint` to this interface, right?

Reply:
Yep, that should do it.

Reply:
I pushed it, but with property `value`, not `type`.